org.tribler.Tribler.yaml

```
@@ -0,0 +1,64 @@
+id: org.tribler.Tribler
+command: tribler
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
```

```suggestion
  - --socket=x11
```

`fallback-x11` should be used only if Wayland is also supported.

---

We probably would need to install the QtWayland module to support Wayland, lets remove the fallback for now.

---

If that app uses Qt, then you should build against the KDE platform instead.